### ISSUE TITLE
Lock FAB Covers Database Settings Text

### ISSUE THREAD

**Issue by christantoan** [Kunzisoft/KeePassDX#1678]
**Describe the bug**

When the "Show lock button" setting is activated, when user scroll down to the bottom of Database settings, the lock button covers the "Maximum size" setting text.

**To Reproduce**

1. Activate "Show lock button" setting.
2. Open Database setting section and scroll down to the bottom 

**Expected behavior**

There are some margin between the lock button and text setting text.

**KeePass Database**

 - N/A.

**KeePassDX:**

 - Version: 4.0.2
 - Build: Free
 - Language: English

**Android:**

 - Device: OnePlus 7 Pro
 - Version: 11

**Additional context**

Screenshots:
![Screenshot_20231106-154043](https://github.com/Kunzisoft/KeePassDX/assets/7300310/897f928e-d30f-45a7-a3f4-a5d3a99325a2)

**Comment by J-Jamet** [comment]
Indeed, I had seen it, but it won't be easy to solve. The problem is that I'm using Android's preference system, which isn't very scalable, so I'd have to redesign it with a custom settings system to add a margin at the end of the page, but that would require a major code overhaul.

**Comment by christantoan** [comment]
Yeah, I often notice this problem on other apps which implement their own custom FAB as well. With official FAB, I think it has been taken into account to create the margin.
